### ISSUE TITLE
[Accessibility][CommandBar]: Disabled Menu items should have aria-disabled=true attribute set

### ISSUE THREAD

**Issue by pankajkhanzode** [OfficeDev/office-ui-fabric-react#1070]
### Bug Report

- __Package version(s)__: (1.7.2)
- __Browser and OS versions__: (all)

#### Describe the issue:

For disabled menu items, aria-disabled attribute should be specified, else screen readers will have trouble reading it clearly. See this: http://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled 

#### Actual behavior:

https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/commandbar
See markup for "disabled" button:
`<button name="Disabled..." disabled="" class="ms-CommandBarItem-link" id="CommandBar983disabled" role="menuitem" aria-haspopup="false" aria-label="Disabled..." icon="Cancel" data-command-key="7"><i class="ms-Icon ms-Icon--Cancel ms-CommandBarItem-icon ms-CommandBarItem-iconColor"></i><span class="ms-CommandBarItem-commandText">Disabled...</span></button>
`
#### Expected behavior:

In addition aria-role="menuitem", set aria-disabled="true" for disabled menu items.

### If applicable, please provide a codepen repro:

Not required.
